ledger/ledger-api-common/src/main/scala/com/digitalasset/platform/common/PlatformTypes.scala

```diff
@@ -12,32 +12,32 @@ import scala.collection.breakOut
 
 object PlatformTypes {
 
-  type GenTransaction[Nid, Cid] = T.GenTransaction[Nid, Cid, V.VersionedValue[Cid]]
+  type GenTransaction[Nid, Cid] = T.GenTransaction.WithTxValue[Nid, Cid]
   val GenTransaction: T.GenTransaction.type = T.GenTransaction
 
-  type GenNode[Nid, Cid] = N.GenNode[Nid, Cid, V.VersionedValue[Cid]]
+  type GenNode[Nid, Cid] = N.GenNode.WithTxValue[Nid, Cid]
 
-  type NodeCreate[Cid] = N.NodeCreate[Cid, V.VersionedValue[Cid]]
+  type NodeCreate[Cid] = N.NodeCreate.WithTxValue[Cid]
   val NodeCreate: N.NodeCreate.type = N.NodeCreate
 
-  type NodeLookupByKey[Cid] = N.NodeLookupByKey[Cid, V.VersionedValue[Cid]]
+  type NodeLookupByKey[Cid] = N.NodeLookupByKey.WithTxValue[Cid]
   val NodeLookupByKey: N.NodeLookupByKey.type = N.NodeLookupByKey
 
   type NodeFetch[Cid] = N.NodeFetch[Cid]
   val NodeFetch: N.NodeFetch.type = N.NodeFetch
 
-  type NodeExercises[Nid, Cid] = N.NodeExercises[Nid, Cid, V.VersionedValue[Cid]]
+  type NodeExercises[Nid, Cid] = N.NodeExercises.WithTxValue[Nid, Cid]
   val NodeExercises: N.NodeExercises.type = N.NodeExercises
 
-  type Event[Nid, Cid] = E.Event[Nid, Cid, V.VersionedValue[Cid]]
+  type Event[Nid, Cid] = E.Event[Nid, Cid, T.Transaction.Value[Cid]]
```

Comment:
I think these aliases are more likely to deal with "transaction value-ness" than "versioned value-ness", though perhaps `Event` _et al_ deserve associated aliases for this, too.